test/system/260-pod-inspect.bats

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "podman pod inspect - format" {
+    skip_if_remote "podman-pod does not work with podman-remote"
```

Comment:
This seems strange. Do you know why?

Cc: @edsantiago @baude

Reply:
This dates to the varlink days: commit 5c108cdab4888c53fbe7d4cb546544cc5a4929aa:
```
looks like 'podman-remote pod' doesn't work; skip test.
```
I haven't tested in the new apiv2 world, but it's probably worth trying to see if things work nowadays.

@boaz0 is there a reason you created a new test file instead of adding this to `200-pod.bats`?

Reply:
I do not remember why I did that but I will move my test there. Thanks :)